crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -80,37 +80,35 @@ impl PackageInfo {
     }
 }
 
-type PackageName = String;
-type PackageVersion = String;
```

Comment:
Do we have to delete `PackageVersion`? That seems fine to keep, no?

Reply:
sure I can bring that back

Reply:
brought it back, but tbh `BTreeMap<String, PackageVersion>` feels more weird to me. the naming conflict feels like a weird reason to elevate one String type over another. Fine with me for this PR though

Reply:
Agreed, but `BTreeMap<String, String>` is very opaque to me. What about `BTreeMap<PackageKey, PackageVersion>`?

Reply:
That's fine with me. I think some of these structs could probably get sorted out a bit